client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -26,11 +26,39 @@ const INSTALL_STATE_COMPLETE = 1;
 const INSTALL_STATE_INCOMPLETE = 2;
 
 export class PaidPlanThankYouCard extends Component {
+	state = {
+		tracksEventSent: false,
```

Comment:
We talked via another channel, but bringing it here for visibility. I don't think we should use component state for this, whether or not to fire a tracks event is independent of the Component render and lifecycle. As such, it's probably better to keep it as an instance property.

Reply:
👍 👍 Thanks! done in https://github.com/Automattic/wp-calypso/pull/33252/commits/933b377f8e50e98554834afa829be522c23f97ac